### ISSUE TITLE
Minor bug on chromium browsers, will be fixed soon

### ISSUE THREAD

**Issue by DamienDavisNeff** [DamienDavisNeff/RedSubscribeButton#17]
If you use the version of this extension on Chromium based browsers via the Chrome Webstore, you may have your settings reset when you switch tabs, due to a typo that was in the released version in the most recent update. It will be a few days for this to pass review before this applies, just be weary of it.

I am not 100% sure if this actually happens or not, but either way the typo has been fixed in Firefox, and on my releases page for this repo. 

TLDR: If you use this extension from the Chrome Webstore, you may or may not have your settings reset when you switch tabs. This issue will be fixed in a few days after passing review.

**Comment by DamienDavisNeff** [comment]
After looking more into it, the typo may just cause issue #13 or #16 to reappear instead.
